src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,92 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+import java.io.IOException;
+
+
+public class Storage {
+    ArrayList<Task> taskList = new ArrayList<>();
+
+    // Overloaded Constructor with filepath
+    public Storage(String filePath) {
+        loadTask(taskList, filePath);
+    }
+
+    public ArrayList<Task> getTaskList() {
+        return this.taskList;
+    }
+
+    // Reusable code for loading task into arrayList
+    public void loadTask(ArrayList<Task> taskList, String filePath) {
+        String path = "src\\main\\java\\data\\duke.txt";
+        path = path.replace("\\", "/");
+        try {
+            File taskFile = new File(path);
+            Scanner reader = new Scanner(taskFile);
+            while (reader.hasNextLine()) {
+                String data = reader.nextLine();
+                if(data.contains("(by:")) {
```

Comment:
There's a few similar to this, but perhaps you could standardise the whitespacing for reserved words? For example, this could be 
```java
if (data.contains("(by:")) {
```

Reply:
Hey @aceszhenwei , Thanks so much for this, I left out many of these whilst doing the branches !